godtools/App/Share/Data/UserCountersRepository/Api/Models/GetUserCountersResponseDecodable.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  GetUserCountersResponseDecodable.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 11/29/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import Foundation
+
+struct GetUserCountersResponseDecodable: Decodable {
```

Comment:
Hey @rachaelblue created this task that will do away with having to create multiple Decodable types when decoding ```data``` attributes.  There's quite a few Decodable types I've created as well.  (https://jira.cru.org/browse/GT-1894)

Reply:
Sounds good!